### PR TITLE
remove archived and deleted data from mvp reports

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?165141

This goes on top of https://github.com/dimagi/commcare-hq/pull/6332 and https://github.com/dimagi/commcare-hq/commit/b567ca265d33b940afa2b25dc11cc9ec08b74e14 is the relevant commit

@biyeun I think you should take a look? or @dannyroberts @benrudolph have also been digging around in mvp. cc @kaapstorm

### CODE REVIEW

**Comment by kaapstorm** [comment]
:thumbsup:

File: custom/_legacy/mvp_docs/tests/test_pillows.py [hunk 1]
```diff
@@ -111,4 +111,15 @@ def test_case_pillow_indicators(self):
         )
         self.assertNotEqual(indicator_case.computed_, {})
 
-
+    def test_delete_doc(self):
```
Comment (biyeun) [inline custom/_legacy/mvp_docs/tests/test_pillows.py]: yessssssss <3

**Comment by biyeun** [comment]
looks good. Should I wait for other eyes or merge?

**Comment by czue** [comment]
I think it's fine to merge. I need to talk to you or @dannyroberts about the best way to deal with reindexing

**Comment by dannyroberts** [comment]
What's that going to entail exactly?

**Comment by czue** [comment]
@dannyroberts I think we need the pillows to process every relevant doc. Any suggestions? I was thinking of naively just resetting the checkpoints, though not sure that's legit.

**Comment by dannyroberts** [comment]
And the net result would be to delete everything of doc_type in `get_deleted_doc_types()` from mvp-indicators db? Wonder if there's a more direct way to do that—maybe by pulling from `'domain/docs'` view using keys in (MVP_DOMAINS × DELETED_DOC_TYPES) and then deleting (or attempting to delete the equivalent doc in mvp-indicators? That'd be faster to run, and have no stale-time, but up to you whether the extra headache is worth it

**Comment by czue** [comment]
yeah that's a good idea. i'll probably take a crack at it tomorrow.

**Pull request merged**